agent/tool-scripts/pcptool

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python3
+# -*- mode: python -*-
+
+import sys
+import os
+import logging
+
+PROG = os.path.basename(sys.argv[0])
+logger = logging.getLogger(PROG)
```

Comment:
Lets drop all the logging and just use `print(f"{PROG}: ...", file=sys.stderr)`, and `sys.exit(1)` instead of `exit(0)` (curious, does it work the way it is?).

Reply:
Yeah it currently works, but I will change it to the suggested method